CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 - Stop using the `typo3/minimal` package on CI (#520, #531)
 - Update to Composer 2.4 (#513)
 - Change the default indentation for rst files to 4 spaces (#194)
+- Replace switchable controller actions with separate plugins (#569)
```

Comment:
This needs to go the the top of the list (as the list is in reverse chronological order).